### PR TITLE
Add ignore option for class names

### PR DESCRIPTION
**Pull request by otacke**
This change adds another option for ignoring nodes for rendering similar to `ignoreTags`.
The new option `ignoreClasses` can hold an array of strings representing class names that should be ignored if they are in a node's class list. This can be desirable in dynamic environments.

### CODE REVIEW

**Comment by otacke** [comment]
[clabot:check]

File: contrib/auto-render/auto-render.js [hunk 1]
```diff
@@ -61,9 +61,24 @@ const renderElem = function(elem, optionsCopy) {
             elem.replaceChild(frag, childNode);
         } else if (childNode.nodeType === 1) {
             // Element node
-            const shouldRender = optionsCopy.ignoredTags.indexOf(
+            let shouldRender = optionsCopy.ignoredTags.indexOf(
                 childNode.nodeName.toLowerCase()) === -1;
```
Comment (ylemkimon) [inline contrib/auto-render/auto-render.js]: IE9 doesn't support `classList`. You can do:
```js
// normalize className to use indexOf for searching
const className = ' ' + childNode.className + ' ';
const shouldRender = optionsCopy.ignoredTags.indexOf(
    childNode.nodeName.toLowerCase()) === -1 &&
    optionsCopy.ignoreClasses.every(
        x => className.indexOf(' ' + x + ' ') === -1)
```

Comment (otacke) [reply contrib/auto-render/auto-render.js]: Thanks for the hint! Didn't know that your support covers this old browser version, too. Will fix it.

Comment (otacke) [reply contrib/auto-render/auto-render.js]: @ylemkimon I changed the code. Thanks again for the hint! I also learned that every is supported by IE9, so no need to do everything "manually" with the for-loops.

**Comment by ylemkimon** [comment]
The default should be an empty array or a null check should be added.

**Comment by otacke** [comment]
I'll gladly add that.

**Comment by otacke** [comment]
@ylemkimon This should do now. Thanks for investing all this time into the project!

**Pull request merged**